canonical-web-and-design/jaas.ai
# Add tutorials

## Done

Add tutorials to site

## Issue / Card

Fixes #511 
## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8029/tutorials
- Click around and check that everything looks and works as expected
- Click through to a tutorial and check that everything looks and works as expected
